variables.tf

```diff
@@ -182,6 +182,22 @@ variable "cdn_host_redirects" {
   default     = []
 }
 
+variable "cdn_url_path_redirects" {
+  description = "CDN FrontDoor url path redirects `[{ \"redirect_type\": \"PermanentRedirect\", \"destination_path\": \"/example\", \"destination_hostname\": \"www.example.uk\", \"operator\": \"Equals\", \"match_values\": [\"/example\"] }]`"
+  type = list(object({
+    redirect_type        = string
+    redirect_protocol    = optional(string)
```

Comment:
If you change these to `optional(string, null)` or `optional(list(string), [])` then you don't need to `lookup()` the values later on

Reply:
Fixed - thanks for highlighting this